src/realm/table_view.cpp

```diff
@@ -664,6 +664,9 @@ bool TableView::is_in_table_order() const
     else if (m_source_column_key) {
         return false;
     }
+    else if (!m_query.m_table) {
```

Comment:
due to setting the `m_table` in every `Results` ctor we need to add this extra check to make sure we don't access a null table on `m_query`